portal/Cargo.toml

```
@@ -14,12 +14,12 @@ version.workspace = true
 [dependencies]
 futures-channel.workspace = true
 futures-util.workspace = true
-oo7 = { workspace = true, features = ["unstable"] }
+oo7 = { workspace = true, default-features = false, features = ["tokio", "native_crypto", "unstable"] }
```

```suggestion
oo7 = { workspace = true, features = ["tokio", "native_crypto", "unstable"] }
```
you set that in the workspace already, also it is weird that the cli uses native crypto & async-std and the portal uses native / tokio, which are the default-features anyway

---

I removed the redundant default-features.